### ISSUE TITLE
Override in component

### ISSUE THREAD

**Issue by sp0ken** [AlekseyPleshkov/nuxt-social-meta#1]
Hi,

Is there a way to override the default meta from within a component?
Similar to what vue-meta does with its `head()` function.

Cheers,

**Comment by LeCoupa** [comment]
[From the source code](https://github.com/AlekseyPleshkov/nuxt-social-meta/blob/master/index.js), it looks he is just overriding `vue-meta`. I guess you should just override the value in a component like you are already doing it.

**Comment by wboka** [comment]
Does anyone have an example of overriding this in a component? I'm at a loss over here.

**Comment by edoardomazzon** [comment]
@wboka Yes, you can override in a pages/component like this (remember ALWAYS the hid property !!) :
```
...
head() {
    return {
      title:
        'Hi, I'm the title',
      meta: [
        {
          hid: 'og:title',
          property: 'og:title',
          content:
            'Hi, I'm the title',
        },
        {
          hid: 'description',
          name: 'description',
          content:
            'Hi, I'm the description',
        },
        {
          hid: 'og:description',
          name: 'og:description',
          content:
            'Hi, I'm the description',
        },
        ...
}
...
```

For the first `title` it will automatically overwrite, while for all the other `meta` properties (such as `og: title`, `og: description`, etc.) the `hid` attribute must be entered to override the desired property.
I hope I was clear and helpful!

**Comment by borisdayma** [comment]
It would be great to have a shortcut to enter the details as per nuxt-social-meta tags (instead of filling each individual field).
